### DISCUSSION TITLE
How translating Korean audio into Japanese text

### DISCUSSION THREAD

**Discussion by aguang1201** [openai/whisper#2105] [Q&A] [answered]
How to translate non-English audio into non-English, such as translating Korean audio into Japanese text?

**Comment by ryanheise** [comment]
Whisper directly supports direct translation of audio only the from language X into English. If you want to target Japanese and still want to use Whisper, you would need to do a two step process where you first do a direct transcription into Korean, and then use a separate model or API (e.g. DeepL, ... etc.) to translate that into Japanese.

**Reply by guijuzhejiang** [reply]
Thanks, loading another model takes up extra VAM, so I'll just use whisper to translate it!

**Comment by JunpyeongKim** [comment, marked as answer]
Hi @aguang1201 
if your input audio is in Korean and you want to translate KR to JP, you could use the following command.
I think the output quality of the large model is better than one of the small model for Korean.

`whisper --model large --language ja input_audio.mp3`

**Reply by guijuzhejiang** [reply]
@JunpyeongKim Thank you,ありがとう,감사합니다,谢谢

**Comment by eguar11011** [comment]
I want to translate a video in English to Spanish
I'm confused how to do it

**Reply by JunpyeongKim** [reply]
Hi @eguar11011 
I believe if your input audio file is in English, you might use this command

`whisper --model <small or large> --language Spanish your_input_audio.mp3`

Thank you.
